sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.NUnit.cs

```diff
@@ -282,6 +282,21 @@ public void TestCaseSource14()
         public void TestCaseSource15() // Don't raise on skipped test methods
         {
         }
+
+        private static readonly TestCaseData[] Cases =
+        {
+          new TestCaseData("foo", "bar").Returns(false),
+          new TestCaseData(100, 100).Returns(true),
+        };
+
+        // See: https://github.com/SonarSource/sonar-dotnet/issues/3751
+        // Returned value is verified by the framework and no assertion is needed.
+        [Test]
+        [TestCaseSource(nameof(Cases))] 
+        public bool Test(object obj1, object obj2) // Noncompliant - FP
```

Comment:
Is there a documentation for the `bool` return type? I've only found `void` examples in the [TestCaseSource docs](https://docs.nunit.org/articles/nunit/writing-tests/attributes/testcasesource.html)

Reply:
Not sure why it does not appear anymore in docs, I've used that in the past and the method is still there in  the latest NUnit version (3.12). 

You can find this mentioned in [2.6.4 docs](https://docs.nunit.org/2.6.4/testCaseSource.html)